python/cugraph/dask/link_analysis/pagerank.py

```diff
@@ -113,15 +129,21 @@ def pagerank(input_graph,
     """
     from cugraph.structure.graph import null_check
 
+    if personalization is not None:
```

Comment:
It was supported in 1D. Have we deprecated all the 1D path? 
If so we should cleanup all 1D files.

Reply:
Personalization is not supported for this release(It will be updated for next). We have deprecated all 1D path as of cython/python and files are updated. In cpp we have a an experimental pathway for 2D currently, I believe we would need to keep the 1D around till experimental moves to main pathway